### ISSUE TITLE
init_module reports as "<unknown>" on Ubuntu

### ISSUE THREAD

**Issue by arossert** [draios/sysdig#992]
I have tried to filter `init_module` syscall but nothing has shown when I used this:
```
sysdig evt.type=init_module
```
After debugging I have used this filter to try to see why:
```
sysdig proc.name=insmod
```
And this is the output:
```
...
27499 15:40:45.491911305 0 insmod (105659) > mmap addr=0 length=5696 prot=1(PROT_READ) flags=2(MAP_PRIVATE) fd=3(<f>/home/arossert/modules/procfs/procfs.ko) offset=0 
27500 15:40:45.491913553 0 insmod (105659) < mmap res=7FB28999A000 vm_size=4492 vm_rss=684 vm_swap=0 
27501 15:40:45.491914503 0 insmod (105659) > <unknown> 
27506 15:40:45.492590310 0 insmod (105659) < <unknown> 
27507 15:40:45.492593203 0 insmod (105659) > munmap addr=7FB28999A000 length=5696 
27508 15:40:45.492597362 0 insmod (105659) < munmap res=0 vm_size=4484 vm_rss=1456 vm_swap=0 
27509 15:40:45.492597949 0 insmod (105659) > close fd=3(<f>/home/arossert/modules/procfs/procfs.ko) 
...
```
It look like it cannot recognise the `init_module` syscall, I have tested it with `delete_module` and it worked fine.
```
...
11829 16:29:55.267990657 3 rmmod (4691) > delete_module 
11893 16:29:55.268672321 3 rmmod (4691) < delete_module 
...
```

Strace output:
```
...
open("/home/arossert/modules/procfs/procfs.ko", O_RDONLY|O_CLOEXEC) = 3
fstat(3, {st_mode=S_IFREG|0664, st_size=5696, ...}) = 0
mmap(NULL, 5696, PROT_READ, MAP_PRIVATE, 3, 0) = 0x7fc878b31000
finit_module(0x3, 0x7fc87892bee9, 0)    = 0
munmap(0x7fc878b31000, 5696)            = 0
close(3)                                = 0
...
```

After debugging this I can see that in `main.c` `TRACEPOINT_PROBE(syscall_enter_probe, struct pt_regs *regs, long id)` the `id` argument is 0 and this is as far as I got (don't know what else I can check).

Tested with `master` and `dev` on Ubuntu 14 and Ubuntu 16 with the same results.
The weird thing is that on RHEL5 it works as expected (`init_module` is recognized).

I will be happy to submit a PR for a fix but I couldn't find what is causing this behavior.
Let me know if more information is required.

**Comment by arossert** [comment]
I have checked the output of strace in RHEL5:
```
read(3, "", 118784)                     = 0
init_module(0x2b295c7ce010, 142928, "") = 0
munmap(0x2b295c7ce000, 266240)          = 0
getpid()                                = 12213
```
Here the syscall is `init_module` and on Ubuntu it is `finit_module` can this be the reason?

**Comment by luca3m** [comment]
I believe so, we need to add `finit_module` to [syscall_table](https://github.com/draios/sysdig/blob/dev/driver/syscall_table.c)

**Comment by arossert** [comment]
@luca3m I will work on that and submit a PR, thanks.

**Comment by arossert** [comment]
Fixed by #996
